jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/RefreshTreeAction.kt

```diff
@@ -17,7 +17,6 @@ class RefreshTreeAction : S3ObjectAction(message("general.refresh"), AllIcons.Ac
     override fun performAction(dataContext: DataContext, nodes: List<S3TreeNode>) {
         val treeTable = dataContext.getRequiredData(S3EditorDataKeys.BUCKET_TABLE)
         val node = nodes.firstOrNull() ?: treeTable.rootNode
-
```

Comment:
Spacing change with no code change?

Reply:
In error, reverting file to original state in next commit